resources/wikia/libraries/jquery/autocomplete/jquery.autocomplete.js

```diff
@@ -129,11 +138,14 @@
     },
 
     fixPosition: function() {
-      var offset = this.el.offset();
-      var parentOffset = $(this.options.appendTo).offset();
-      var el = $(this.options.appendTo).find('#' + this.mainContainerId).css({ top: (offset.top + this.el.innerHeight() - parentOffset.top) + 'px', left: (offset.left - parentOffset.left) + 'px' });
-      if ( this.options.positionRight !== null ) {
-        el.css({ right: this.options.positionRight });
+      // Wikia change - setPosition option
+      if (this.options.setPosition) {
+        var offset = this.el.offset();
+        var parentOffset = $(this.options.appendTo).offset();
+        var el = $(this.options.appendTo).find('#' + this.mainContainerId).css({ top: (offset.top + this.el.innerHeight() - parentOffset.top) + 'px', left: (offset.left - parentOffset.left) + 'px' });
+        if ( this.options.positionRight !== null ) {
+          el.css({ right: this.options.positionRight });
+        }
```

Comment:
The change is adding the `if (this.options.setPosition) {}` wrapper to be avoid setting the position by the script itself. Would changing the comment to be more descriptive fix the problem?